docs/rfcs/managed.md

```diff
@@ -0,0 +1,150 @@
+# Managed CRD
+
+**Status:** *Draft*
+
+**Author:** Antoine Toulme (@atoulme)
+
+**Date:** 2025-03-20
+
+## Objective
+
+Condense all best practices and deployment patterns of OpenTelemetry components into a single custom resource
+that covers the use cases fulfilled by the OpenTelemetry helm charts.
+
+## Summary
+This request for comment aims to establish a new approach to management of resources in Kubernetes clusters,
+with the explicit goal of taking over the whole cluster, and making decisions
+on how to best instrument all resources in the cluster on behalf of the administrator based on best practices,
+OpenTelemetry standards and semantic conventions.
+
+The operator project offers custom resources (CR) to allow Kubernetes users/administrators to define OpenTelemetry components
+in their Kubernetes environments. Customers however do not want to manage components themselves, especially if they need
+to think about component versioning and upgrade patterns, and would rather have an out-of-the-box experience that 
+requires little insight and gets them to a working state effortlessly.
+
+## Goals and non-goals
+
+### Give a comprehensive, out-of the box experience of OpenTelemetry for a first-time user
+The goal is to delight first-time users by giving them a complete view of what is possible with OpenTelemetry.
+
+### Do not offer a way to turn on and off specific features
+We mean to offer a specific functional coverage of a specific use case that offers best practices of OpenTelemetry.
+
+We don't allow toggling features specifically because we want to avoid increasing the need for functional test coverage.
+
+If customers want to increase their level of control of the feature set, they can remove the managed custom resource and instead
+follow the traditional route of setting up collectors, target allocators, instrumentation custom resources as is possible today.
+
+### OTLP, single endpoint support
+
+The solution exports data to a single endpoint using the OTLP protocol. Customers may choose to deploy a gateway
```

Comment:
```suggestion
The solution exports data to a single endpoint using the OTLP protocol. Users may choose to deploy a gateway
```

by my count there are still ten instances of customer

Reply:
oops fixed.